src/index.ts

```diff
@@ -150,6 +159,91 @@ export class PaddleSdk<TMetadata = any> {
     }
   }
 
+  private parsePaymentSucceededWebhookEvent(
+    body: RawPaddlePaymentSucceededAlert
+  ): PaddleSdkPaymentSucceededEvent<TMetadata> {
+    return {
+      // EVENT ---
+
+      eventType: PaddleSdkWebhookEventType.PAYMENT_SUCCEEDED,
+      eventId: convertApiInteger(body.alert_id),
+      eventTime: convertApiDate(body.event_time, 'DATE_TIME'),
+
+      // ORDER ---
+
+      metadata: this.parseMetadata(body.passthrough),
+      orderId: body.order_id,
+      checkoutId: body.checkout_id,
+      coupon: body.coupon,
+      receiptUrl: body.receipt_url,
+      productId: body.product_id,
```

Comment:
Parse as an integer, for consistency with subscription hooks
```suggestion
      productId: convertApiInteger(body.product_id),
```